mne/io/edf/tests/test_edf.py

```diff
@@ -70,6 +72,14 @@ def test_bdf_data():
     assert_true((raw_py.info['chs'][63]['loc']).any())
 
 
+def test_edf_overlapping_annotations():
+    n_warning = 2
+    with warnings.catch_warnings(record=True) as w:
+        read_raw_edf(edf_overlap_annot_path, verbose=True)
```

Comment:
use preload=True

travis says "RuntimeError: EDF+ Annotations (TAL) channel needs to be parsed completely on loading. You must set preload parameter to True."

was the test running for you locally?